_components/input-mask/guidance/implementation.md

```diff
@@ -1,24 +1,90 @@
-The examples in the Preview section, above, are common input masks, but you can adapt the code to meet the needs of your project. In order to extend the functionality of the input mask, you must include a `placeholder` attribute as the visible input mask, using the underscore `_` to represent a character the user should enter. You may use punctuation literals like `[space]`, `-`, `/`, or `(` to help format your mask, but these literals will not be included in the final submitted value. You also need a related `pattern` attribute on the `input` element, its value epxressed as a regular expression that describes the intended format of the input data. Additionally, if the expected input is numeric, make sure to add the `inputmode="numeric"` attribute and setting so the numeric keypad is displayed. For example, an `input` with an input mask of three groups of three numerals, separated with spaces might have the following properties: 
+{:.usa-content-list }
+The examples in the Preview section above are common input masks, but you can adapt the code to meet the needs of your project. In order to extend the functionality of the input mask, you must include a `placeholder` attribute as the visible input mask, using the underscore `_` to represent a character the user should enter. You may use punctuation literals like `[space]`, `-`, `/`, or `(` to help format your mask, but these literals will not be included in the final submitted value. _See a complete list of format characters below_.
+
+{:.usa-content-list }
+- **Add component classes.** The structure should include a base element with the class `usa-input-mask`. Inside of that base element there should be an input element with the class `usa-input-mask__field` and a message element (span or div) with the class `usa-input-mask__message`.
+- **Add a `mask` attribute to the input element.** The mask attribute is used to define the format of your mask. Use `9`s for numbers and `A`s for letters. For example, a typical SSN number format in the mask attribute should look like `999-99-9999`.
```

Comment:
Does it make sense to list out all of the component attributes in a properties/procedures section like we do for components like [time picker](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/mm-input-mask-guidance/components/time-picker/)? 
Asking because there are a fair number of attributes to account for, and it is a little more clear to see them in a tabular format and grouped together.

Reply:
@amyleadem This is complete and is looking so much better! https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/mm-input-mask-guidance/components/input-mask/